renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["github>ministryofjustice/hmpps-renovate-config:base"],
+  "enabledManagers": ["dockerfile", "docker-compose", "helmv3", "helm-values", "circleci", "github-actions"],
```

Comment:
Decided to not extend https://github.com/ministryofjustice/hmpps-renovate-config/blob/main/node.json because this project has issues with many npm dependencies and didn't seem worth the effort to continually review and tweak Renovate PRs